openid/src/main/java/fish/payara/security/openid/controller/ConfigurationController.java

```diff
@@ -365,6 +364,45 @@ private void validateConfiguration(OpenIdConfiguration configuration) {
         return errorMessages;
     }
 
+    /**
+     * Create Url query from pairs of parameters.
+     */
+    public String createUrlQuery(String[] parameters) {
+        StringBuilder extraParametersFromAnnotationBuf = new StringBuilder();
+        String extraParamDelim = "";
+        for (String extraParameter : parameters) {
+            String[] parts = extraParameter.split("=");
+            String key = parts[0];
+            String value = parts.length > 1 ? parts[1] : null;
+            extraParametersFromAnnotationBuf.append(extraParamDelim)
+                    .append(URLEncoder.encode(key, StandardCharsets.UTF_8))
+                    .append("=")
+                    .append(URLEncoder.encode(value, StandardCharsets.UTF_8));
```

Comment:
My in-head JVM cannot determine how exactly `null` value gets encoded here.

I quick look into javadoc says it's encoded by [throwing](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/net/URLEncoder.html#encode(java.lang.String,java.nio.charset.Charset))  a `NullPointerException`. 

A unit test for this and next method would be helpful. (make them default-access and static)